isce2_topsapp/topsapp_proc.py

```diff
@@ -50,6 +59,10 @@ def topsapp_processing(*,
     with open(TEMPLATE_DIR/'topsapp_template.xml', 'r') as file:
         template = Template(file.read())
 
+    geocode_list = GEOCODE_LIST_BASE.copy()
+    if estimate_ionosphere_delay:
+        geocode_list.append('merged/topophase.ion')
```

Comment:
@cmarshak just to double check if this is the unwrapped topophase or the wrapped variant. 
Check if tehre is not a topophase.ion.unw or so

Reply:
Let's look at the products together. I was going to just include this raster in the product.